src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
you can't set margins to BEM block

Reply:
if you need to reset default margins for the elements, you can create reset.css file for this purpose